docs/collection-trait.md

```diff
@@ -95,6 +95,10 @@ $collection->toArray();
 
 Internally it is call the `ArrayIterator::append` and returning the instance to allow fluent calls.
 
+## chunk
+
+This method [mirrors the behavior of `array_chunk`](https://www.php.net/manual/function.array-chunk.php) and returns a zero indexed numeric array of the current Collection based on the chunk size provided.
```

Comment:
This comment is the same of the interface and is not giving any detail of the implementation done in the trait (and also the same "Collection With Uppercase On The First Letter")

Reply:
Will update!